### DISCUSSION TITLE
反無人機顯學，法國推出新型自動防空機炮塔

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#674] [General]
終究科技的進步是因為戰爭 希望不要真的用的上
https://technews.tw/2022/10/24/thales-and-nexter-unveiled-latest-rapidfire-anti-drone-auto-turret/

**Comment by bww0401** [comment]
我覺得科技應該是要用以增進人類福祉，如果真的拿來當作武器，那就太恐怖也太可惜了

**Comment by alvin1226** [comment]
越來越有86的感覺了，最後國家之間的戰爭是在比誰的機器人多，不用派人類出擊，好像也不錯？

**Comment by liangyinglly** [comment]
雖然戰爭依照以前的經驗，總是能造成技術與知識的大躍進，但還是希望大家都能繼續過著和平穩定的日子

**Comment by Valentinaa0214** [comment]
雖然AI用於戰爭也有一些好處，像是透過精確定位來減少戰爭中的傷亡，但可能會出現難以掌控的人與人、人與機器、機器與機器之間相互交戰的危機。原本用於造福人類社會的科技發明被用於戰爭用途，是相當不可取的，希望國際能正視這個問題。

**Comment by Vincent20030314** [comment]
雖然機器人戰爭可以不用犧牲生命，但可能導致戰爭更加頻繁，可以的話希望還是不要為了戰爭而發展科技比較好。

**Comment by jerry0108566** [comment]
要機器人打仗不如開一把CS上去輸贏

**Comment by yjpxi** [comment]
近年來，隨著國際社會對和平與合作的重視增強，許多科技創新正在專注於解決全球性的問題，如氣候變化、公共衛生、能源安全等。這些領域的進步不僅有助於避免戰爭，更能促進全球的穩定與繁榮。
